src/framework/uicomponents/qml/MuseScore/UiComponents/ValueAdjustControl.qml

```diff
@@ -9,6 +9,8 @@ Item {
     signal increaseButtonClicked
     signal decreaseButtonClicked
 
+    property int timerTimeInterval: 300
```

Comment:
Because ValueAdjustControl is used in only one place, I think it is better to make this directly the correct value (100), instead of setting it to a "wrong" default value that is never used and setting it separately in IncrementalPropertyControl.

Reply:
Yes, this makes more sense.